src/plugins/newsfeed/public/components/newsfeed_header_nav_button.tsx

```diff
@@ -66,14 +66,10 @@ export const NewsfeedNavButton = ({ apiFetchResult }: Props) => {
           aria-expanded={flyoutVisible}
           aria-haspopup="true"
           aria-label="Newsfeed menu"
+          notification={showBadge ? true : null}
           onClick={showFlyout}
         >
           <EuiIcon type="cheer" size="m" />
-          {showBadge ? (
-            <EuiNotificationBadge className="euiHeaderNotification" data-test-subj="showBadgeNews">
```

Comment:
I'm guessing some tests will need to be updated to figure out how to get around not having a DTS on the notification.

Reply:
Although there was a DTS, there was no actual test using it... searching for this string turns up no other results. Perhps there should be a test, but that's outside my personal wheelhouse.